ui/pages/swaps/awaiting-swap/awaiting-swap.js

```diff
@@ -192,7 +181,11 @@ export default function AwaitingSwap({
 
     if (!trackedQuotesExpiredEvent) {
       setTrackedQuotesExpiredEvent(true);
-      quotesExpiredEvent();
+      metaMetricsEvent({
```

Comment:
[req] I know this is pre-existing but can we rename metaMetricsEvent with trackEvent

Reply:
@brad-decker Done [here](https://github.com/MetaMask/metamask-extension/pull/13955/commits/068c92dafb34adb2277d7b9eae602fe9ab9c42b1)